server/src/main/java/org/opensearch/rest/action/RestCancellableNodeClient.java

```diff
@@ -83,7 +83,7 @@ public static int getNumChannels() {
     /**
      * Returns the number of tasks tracked globally.
      */
-    static int getNumTasks() {
+    public static int getNumTasks() {
```

Comment:
Let's keep this value sealed please:

```suggestion
    static int getNumTasks() {
```

Reply:
`RestCancellableNodeClient` is public API, we should not open up methods there just for tests, we could expose this check in test case scaffolding fe (see please `OpenSearchTestClusterRule::clearClusters` as an example), without modification in APIs